nilearn/decoding/space_net.py

```diff
@@ -1134,6 +1131,27 @@ def _binarize_y(self, y):
         self.n_classes_ = len(self.classes_)
         return y
 
+    def score(self, X, y, sample_weight=None):
+        """Returns the mean accuracy on the given test data and labels.
+
+        Parameters
+        ----------
+        X : list of Niimg-like objects
+            See http://nilearn.github.io/building_blocks/
```

Comment:
By the way the full URL should be: http://nilearn.github.io/building_blocks/manipulating_mr_images.html#niimg. Can you put it on one line? That will go over the PEP8 width limit but that is what we are doing everywhere else.